test/extended/util/annotate/annotate.go

```diff
@@ -35,6 +37,28 @@ func main() {
 		os.Exit(1)
 	}
 
+	// All tests must be associated with a sig (either upstream), or downstream
+	// If you get this error, you should add the [sig-X] tag to your test (if its
+	// in origin) or if it is upstream add a new rule to rules.go that assigns
+	// the test in question to the right sig.
+	//
+	// Upstream sigs map to teams (if you have representation on that sig, you
+	//   own those tests in origin)
+	// Downstream sigs: sig-imageregistry, sig-builds, sig-devex
+	var errors []string
+	for from, to := range generator.output {
+		if !reHasSig.MatchString(from) && !reHasSig.MatchString(to) {
```

Comment:
why do we care about the from?

Reply:
`from` contains strings that `to` doesn't.  The previous bug fixed the issue where we were doubling up the first part of the from, now we can't just search to (that's everything on the test and that we added), we have to search from (includes things we won't change)

Reply:
ok

Reply:
i was assuming to would be the full new string, not just the additions